### PR TITLE
Fix coding standards compliance issues in `config`

### PR DESCRIPTION
**Pull request by sjinks**
## Description

This PR fixes code style compliance issues found by phpcs.

### Fix code style compliance issues in `config` directory

  * Fix issues found by phpcs;
  * Add phpcs:ignore where appropriate to suppress false positives or known good code.

## Checklist

Please make sure the items below have been covered before requesting a review:

- [x] This change works and has been tested locally (or has an appropriate fallback).
- [ ] This change works and has been tested on a Go sandbox.
- [ ] This change has relevant unit tests (if applicable).
- [ ] This change has relevant documentation additions / updates (if applicable).
- [x] I've created a changelog description that aligns with the provided examples.

## Steps to Test

Make sure that the CI passes.

### CODE REVIEW

File: config/class-site-details-index.php [hunk 1]
```diff
@@ -145,9 +145,9 @@ public function get_jetpack_info() {
 
 		if ( class_exists( 'Jetpack' ) ) {
 			$jetpack_info['available'] = true;
-			$jetpack_info['active'] = \Jetpack::is_active();
-			$jetpack_info['id'] = \Jetpack::get_option( 'id' );
-			$jetpack_info['version'] = JETPACK__VERSION;
+			$jetpack_info['active']    = \Jetpack::is_active();
```
Comment (sjinks) [inline config/class-site-details-index.php:148]: `\Jetpack::is_active()` is deprecated but I have no idea what to replace it with.

**Review by rebeccahum — APPROVED** [review]
🚢

**Comment by sjinks** [comment]
@pauarge do you happen to remember which one? :-) This branch is now even with `master`, there are no changes :-) - https://github.com/Automattic/vip-go-mu-plugins/pull/2470/files

**Comment by sjinks** [comment]
Disregard that, I have found it.

**Pull request closed without merging**